code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -59,18 +59,19 @@
 
 /obj/item/gun/energy/gun/blueshield
 	name = "advanced stun revolver"
-	desc = "An advanced stun revolver with the capacity to shoot both electrodes and lasers."
+	desc = "An advanced stun revolver with the capacity to shoot both disablers and lasers."

```

Comment:
Is the sprite also changed?

Reply:
Yeah, I made new sprites for these using the same pallets as other disablers.

Reply:
Can ya add the sprite changes to the PR description?

Reply:
Sure!